challenges/classes.js

```diff
@@ -1,7 +1,50 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+/* == Step 1: Base Constructor ==
+  Create a constructor function named CuboidMaker that accepts properties for length, width, and height
+*/
+class cuboidMAker{
```

Comment:
make sure you check for typos, your code won't run as is, but ran fine once i corrected them